sbysrc/sby_mode_bmc.py

```diff
@@ -24,10 +24,21 @@ def run(task):
     task.handle_int_option("append", 0)
     task.handle_str_option("aigsmt", "yices")
 
-    for engine_idx, engine in task.engine_list():
+    for engine_idx, engine_section in task.engine_list():
+        if isinstance(engine_section, list):
+            engine = engine_section
+            engine_name = None
+        else:
+            engine = engine_section[1][0]
```

Comment:
I ran into an indexing error with that as well but this was part of the fix, my initial test case for it might not have been enough, can you provide the sby file you were using to test?

Reply:
It was `tests/unsorted/2props1trace.sby`, I only added ` bmc` to the existing `[engines]` section.